co-authors-plus.php

```diff
@@ -1143,7 +1147,7 @@ public function fix_author_page( $selection ) {
 		}
 
 		$author = $this->get_coauthor_by( 'user_nicename', $author_name );
-
+		
```

Comment:
Looks like unnecessary extra whitespace was added here.

Reply:
My bad. I just removed the accidentally added spacing.